src/System.Text.Json/tests/Serialization/TestClasses.SimpleTestClass.cs

```diff
@@ -122,7 +122,7 @@ public class SimpleTestClass : ITestClass
                 @"""MyDateTime"" : ""2019-01-30T12:01:02.0000000Z""," +
                 @"""MyDateTimeOffset"" : ""2019-01-30T12:01:02.0000000+01:00""," +
                 @"""MyGuid"" : ""1B33498A-7B7D-4DDA-9C13-F6AA4AB449A6""," +
-                @"""MyUri"" : ""https:\/\/github.com\/dotnet\/corefx""," +
```

Comment:
Since this string/class is used both for reading and writing, we are losing _some_ coverage on the read side by removing these, where we should successfully deserialize payloads containing such escaped text, but I don't think it was the original intention of this test. I think this is fine, and we should probably add a test specific to deserializing such escaped json.

Reply:
There exists such a test already: https://github.com/dotnet/corefx/blob/0aa9f70b84674f27d18172058406aa03f1572526/src/System.Text.Json/tests/Serialization/Value.ReadTests.cs#L291